.changes/unreleased/NOTES-20250414-095537.yaml

```
@@ -0,0 +1,5 @@
+kind: NOTES
+body: 'This alpha pre-release adds support for managed resource identity, which can be used with Terraform v1.12.0-beta1. Acceptance tests can use the `ImportBlockWithResourceIdentity` kind to exercise the import of a managed resource using its resource identity object values instead of using a string identifier.'
```

I think we can just go ahead and release a beta at this point, aligning with core (indicating our general confidence in the approach)

```suggestion
body: 'This beta pre-release adds support for managed resource identity, which can be used with Terraform v1.12.0-beta1. Acceptance tests can use the `ImportBlockWithResourceIdentity` kind to exercise the import of a managed resource using its resource identity object values instead of using a string identifier.'
```